routes/chars.js

```diff
@@ -4,17 +4,130 @@ var express = require('express'),
     importChar = npmarmory.importChar,
     Char = npmarmory.Char;
 
-// let's see if we can break node here.  ._. prelim logic
-// to import my monk
+// defaults
+var jadeRegionDefault = "US",
+  jadeNameDefault = "Caligraphy",
+  jadeRealmDefault = "Windrunner";
+
+// pre-declare empty so we know where values came from
+var jadeRegion, jadeName, jadeRealm;
+
+var charImporting = function(req, res) {
+
+  //catch tmps from param or POST
+  var tmpRealm = req.tmpRealm, 
+    tmpRegion = req.tmpRegion,
+    tmpName = req.tmpName;
+
+  // if param exist overwrite defaults
+  // I think only (tmp != "undefined") works too
+  if(tmpRegion && tmpRegion != "" && tmpRegion != "undefined"){
```

Comment:
So, for setting defaults, the way you want to do it as such:

var region = postRegion || "US";

So the region returned from the _POST data goes first, and then just use an OR (||) and add the defaulted value afterward.

Since it's been so long, I'm going to pull this branch and edit it to go ahead and do it that way. I'll also look into fixing the OH issue tonight.